packages/api/src/database/migrations/23_user_change_eth_fields.ts

```diff
@@ -0,0 +1,39 @@
+import { PraiseModel } from '@/praise/entities';
+import { UserModel } from '@/user/entities';
+
+const up = async (): Promise<void> => {
+  const users = await UserModel.find();
+
+  if (users.length === 0) return;
+
+  const updates = await Promise.all(
+    users.map((s) => ({
+      updateOne: {
+        filter: { _id: s._id },
+        update: {
+          $set: {
+            rewardsEthAddress: s.ethereumAddress,
+            identityEthAddress: s.ethereumAddress,
```

Comment:
`s.ethereumAddress` don't exist since it is removed from `UserDocument`. Ok I think to use type `any` in the migration. 

```
rewardsEthAddress: (s as any).ethereumAddress,
identityEthAddress: (s as any).ethereumAddress,
```